mods/ts/metrics.yaml

```diff
@@ -3,6 +3,7 @@ Metrics:
 	ColorPickerActorType: mmch.colorpicker
 	ColorPickerRemapIndices: 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31
 	TextfieldColorHighlight: 1a1a1a
+	TextBackgroundColorSelected: 1a1a1a
```

Comment:
We can go for a color that works on all backgrounds to avoid duplicating `lobby.yaml`. Something around `#777777` seems to be OK.

Reply:
Would we want to do the same for the `Textfield` since they have selection functionality aswell and we'd want matching backgrounds, no?

I'm assuming this is so that changes to the lobby can be made across all mods without having to change many files?

Reply:
Yes, the same color should be applied to the textfield selection. 

TS, RA and D2k share many of the chrome layout files (found in the common folder). TD is the odd one out as it implements a different UI (that is probably for historical reasons but it also serves as an example of how to make an alternative UI). Duplicating the chrome yamls (with such a small change) is a maintenance burden.